packages/table/src/Cell/Cell.styles.ts

```diff
@@ -0,0 +1,92 @@
+import { TransitionStatus } from 'react-transition-group';
+
+import { css } from '@leafygreen-ui/emotion';
+import { spacing, transitionDuration } from '@leafygreen-ui/tokens';
+
+import { Align } from './HeaderCell/HeaderCell.types';
+
+/** The base left & right padding in the table */
+export const baseTableSidePadding = spacing[4];
+
+/** the default width of the expand icon */
+const iconSize = 28;
+
+export const baseCellStyles = css`
+  padding: 0 8px;
+  overflow: hidden;
+  text-overflow: ellipsis;
+
+  &:focus-visible {
+    box-shadow: inset;
+  }
+  &:last-child {
+    padding-right: ${baseTableSidePadding}px;
+  }
+`;
+
+const flexAlignment: Record<string, string> = {
+  left: 'start',
+  right: 'end',
+  center: 'center',
+};
+
+export const alignmentStyles = (align: Align = 'left') => css`
```

Comment:
`Align = Align.Left?`

Reply:
`Align` is `HTMLElementProps<'td'>['align']`, so that won't work